testing/tests/ui/iso646.rs

```diff
@@ -0,0 +1,46 @@
+use rinja::Template;
```

Comment:
Thanks for adding the ui tests for error output!

Reply:
I love UI tests, and I love that it's so simple to do them in rust. In most languages you only test the "good" outcomes, but the bad outcomes are just as important.